### PR TITLE
[Added] [BEL-5579] Add investment endpoints

### PR DESCRIPTION
**Pull request by AlbertMaristany**
:question: What
---
- Added support for `investments/portfolios` and `investments/transactions` endpoints
- Refactored endpoints namespaces to allow calls to `/api/` and `/investments/` endpoints

:speech_balloon: Why
---
- Allow our SDKs to have complete coverage of API endpoints

[BEL-5579](https://belvoteam.atlassian.net/browse/BEL-5579)

### CODE REVIEW

File: lib/belvo.rb [hunk 1]
```diff
@@ -121,5 +121,17 @@ def institutions
     def widget_token
       @widget_token = WidgetToken.new @session
     end
+
+    # Provides access to InvestmentsPortfolio resource
+    # @return [InvestmentsPortfolio]
+    def investments_portfolio
+      @widget_token = InvestmentsPortfolio.new @session
```
Comment (benitocas88) [inline lib/belvo.rb:128]: should be `@investments_portfolio = InvestmentsPortfolio.new @session` ?

File: lib/belvo.rb [hunk 2]
```diff
@@ -121,5 +121,17 @@ def institutions
...
+    end
+
+    # Provides access to InvestmentsTransaction resource
+    # @return [InvestmentsTransaction]
+    def investments_transaction
+      @widget_token = InvestmentsTransaction.new @session
```
Comment (benitocas88) [inline lib/belvo.rb:134]: ⬆️

**Review by benitocas88 — DISMISSED** [review]
🚀 gonna approve in order to unblock you, just two comments :)

File: lib/belvo.rb (hunk 1 above)
Comment (AlbertMaristany) [reply lib/belvo.rb:128]: :man_facepalming: 
You know, copy pasting...

**Review by menecio — APPROVED** [review]

**Pull request merged**